lms/djangoapps/instructor/utils.py

```diff
@@ -186,7 +187,13 @@ def collect_ora2_data(course_id, include_email=False):
     cursor.execute(raw_queries[0])
     cursor.execute(raw_queries[1], [course_id])
     header = [item[0] for item in cursor.description]
-    return header, cursor.fetchall()
+    header.append('formatted_answer')
+    raw_answer_index = 4
```

Comment:
I'm really not a fan of this, but I'm not sure how else to do this. The query upon which this function relies returns the raw_answer as the fifth column (thus index 4).